docs/llm/reference.md

```diff
@@ -12,8 +12,27 @@ It is now integrated into OpenVINO Model Server providing efficient way to run g
 
 Check out the [quickstart guide](quickstart.md) for a simple example that shows how to use this feature.
 
+## Servable Types
+
+Starting with 2025.1, we can highlight four servable types. Such distinction is made based on the input type and underlying GenAI pipeline.
+The servable types are:
+- Language Model Continuous Batching,
+- Language Model Stateful,
+- Visual Language Model Continuous Batching,
+- Visual Language Model Stateful.
+
+First part - Language Model / Visual Language Model - determines whether servable accepts only text or both text and images on the input.
+Seconds part - Continuous Batching / Stateful - determines what kind of GenAI pipeline is used as the engine. By default CPU and GPU devices work on Continuous Batching pipelines. NPU device works only on Stateful servable type.
+
```

Comment:
It should be shortly described what is the difference between continuous batching and stateful pipelines. Include a references. Most importantly stateful pipeline can process the requests sequentially only. Continuous batching can increase efficiently of parallel requests.

Reply:
Not sure if it's relevant to the user. Sequential processing is mentioned in limitations section which implies efficient parallel processing is a default. Here I only wanted a simple explanation why we even have such distinction and I tried to limit the content to the minimum.